docs/flocker-features/emc-coprhd-configuration.rst

```diff
@@ -0,0 +1,14 @@
+.. _emc-coprhd-backend:
+
+=================================
+ EMC CoprHD Block Device Backend
+=================================
+
+.. begin-body
+
+CoprHD-Flocker integration enables CoprHD to provide persistent storage to Docker containers via Flocker.
```

Comment:
Add "CoprHD is an open source software defined storage controller and API platform. It enables policy-based management and cloud automation of storage resources for block, object and file storage providers." as the first sentence.